### PR TITLE
Add docs for coupon credits

### PR DESCRIPTION
**Pull request by aayushmahapatra**

### CODE REVIEW

File: pages/billing/credits.mdx [hunk 1]
```diff
@@ -0,0 +1,31 @@
+import { Callout } from "nextra-theme-docs";
+import Navigation from "components/navigation";
+
+# Coupon Credits
+
+Spheron credits reflect our commitment to provide users with enhanced payment flexibility and streamlined transactional processes.
+One of the standout attributes of the Spheron Credits is their universal applicability.
+Single credit activation code can be deployed across multiple organizations on our platform.
+
+You can use coupon credits to add bonuses for free.
```
Comment (rekpero) [inline pages/billing/credits.mdx:10]: not for free bro, it'll deduct from credit

**Review by rekpero — CHANGES_REQUESTED** [review]

File: pages/billing/credits.mdx [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+import Navigation from "components/navigation";
+
+# Coupon Credits
+
+You can use coupon credits to buy bonuses.
+
+1. Navigate to the "Coupon Credits" section under the **Billing** tab.
+2. Click "Add Credit".
+3. Enter the credit coupon code, complete the CAPTCHA verification and Click "Redeem Credit".
+4. You can view the status of all your credits here, categorized as Active, Pending, and Expired.
```
Comment (rekpero) [inline pages/billing/credits.mdx:11]: mention how to use these credits as well. like add upgrade plan with credits, add ad-ons with credits and topup compute credit with credits

Comment (rekpero) [reply pages/billing/credits.mdx:11]: add these section and then we are good

**Review by rekpero — CHANGES_REQUESTED** [review]

File: pages/billing/credits.mdx [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+
+## Upgrading Plans with Credits
+
+1. Navigate to the "My Plan" section under the **Billing** tab.
+2. Click "Change or upgrade plan" and Select your desired plan.
+3. Choose **Spheron Credit** in Select Payment Method.
+4. Read the transaction details and Click "Deposit." If you have already deposited the funds, you can move to **Step 8**.
+5. To make sure your subscription is renewed every month, you need to deposit funds to the Spheron escrow wallet.
+6. Add the amount you want to approve and Click "Approve." This amount should be more than or equal to the plan's Token-Value (total amount).
+7. Add the amount you want to deposit and Click "Deposit." Make sure it is not more than the approved amount, or else the transaction will fail.
```
Comment (rekpero) [inline pages/billing/credits.mdx:37]: why is this step needed for credit payment? did you not tried it yourself??? Credit shouldn't need user to deposit any token to escrow wallet

File: pages/billing/credits.mdx [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+8. Click "Upgrade Plan" to initiate the transaction. Upon successful transaction, the plan will be upgraded to Pro.
+
+## Adding Add-ons with Credits
+
+1. Navigate to the "Add-on" section under the **Billing** tab.
+2. Click "Buy Add-on" and Select the type of Add-on and the Add-on Amount.
+3. To add more add-on fields, Click "Add Add-on Field".
+4. After adding all the add-on fields, Click "Review."
+5. Choose **Spheron Credit** in Select Payment Method.
+6. Read the transaction details, and Click "Add" to initiate the transaction.
```
Comment (rekpero) [inline pages/billing/credits.mdx:47]:
```suggestion
6. Read the add-on's payment details, and Click "Add" to initiate the payment.
```

File: pages/billing/credits.mdx [hunk 5]
```diff
@@ -0,0 +1,60 @@
...
+7. Upon successful transaction, the Add-on will be added.
+
+## Topping up Compute Credits with Credits
+
+1. Navigate to the "Compute Usage" section under the **Billing** tab.
+2. Click "Top Up" under **Balance** and Select `Compute Credits` in Add-on and the Amount you wish to add.
+3. To add more add-on fields, Click "Add Add-on Field".
```
Comment (rekpero) [inline pages/billing/credits.mdx:54]: Remove this as this is not possible. there is no other add-on fields on the compute anyways

File: pages/billing/credits.mdx [hunk 6]
```diff
@@ -0,0 +1,60 @@
...
+4. After adding all the add-on fields, Click "Review."
+5. Choose **Spheron Credit** in Select Payment Method.
+6. Read the transaction details, and Click "Add" to initiate the transaction.
+7. Upon successful transaction, the Add-on will be added.
+
+## Topping up Compute Credits with Credits
+
+1. Navigate to the "Compute Usage" section under the **Billing** tab.
+2. Click "Top Up" under **Balance** and Select `Compute Credits` in Add-on and the Amount you wish to add.
```
Comment (rekpero) [inline pages/billing/credits.mdx:53]:
```suggestion
2. Click "Top Up" under **Balance** and Select `Compute Credits` in Add-on and the Amount you wish to add and click on Review.
```

File: pages/billing/credits.mdx [hunk 7]
```diff
@@ -0,0 +1,60 @@
...
+3. To add more add-on fields, Click "Add Add-on Field".
+4. After adding all the add-on fields, Click "Review."
```
Comment (rekpero) [inline pages/billing/credits.mdx:55]: remove this step

File: pages/billing/credits.mdx [hunk 8]
```diff
@@ -0,0 +1,60 @@
...
+5. Choose **Spheron Credit** in Select Payment Method.
+6. Read the transaction details, and Click "Add" to initiate the transaction.
```
Comment (rekpero) [inline pages/billing/credits.mdx:57]:
```suggestion
6. Read the add-on's payment details, and Click "Add" to initiate the payment.
```

File: pages/billing/credits.mdx [hunk 9]
```diff
@@ -0,0 +1,60 @@
...
+7. Upon successful transaction, the Add-on will be added.
```
Comment (rekpero) [inline pages/billing/credits.mdx:58]:
```suggestion
7. Upon successful payment, the Add-on will be added.
```

File: pages/billing/credits.mdx [hunk 10]
```diff
@@ -0,0 +1,60 @@
...
+
+## Adding Add-ons with Credits
+
+1. Navigate to the "Add-on" section under the **Billing** tab.
+2. Click "Buy Add-on" and Select the type of Add-on and the Add-on Amount.
+3. To add more add-on fields, Click "Add Add-on Field".
+4. After adding all the add-on fields, Click "Review."
+5. Choose **Spheron Credit** in Select Payment Method.
+6. Read the transaction details, and Click "Add" to initiate the transaction.
+7. Upon successful transaction, the Add-on will be added.
```
Comment (rekpero) [inline pages/billing/credits.mdx:48]:
```suggestion
7. Upon successful payment, the Add-on will be added.
```

File: pages/billing/credits.mdx [hunk 11]
```diff
@@ -0,0 +1,60 @@
...
+## Upgrading Plans with Credits
+
+1. Navigate to the "My Plan" section under the **Billing** tab.
+2. Click "Change or upgrade plan" and Select your desired plan.
+3. Choose **Spheron Credit** in Select Payment Method.
+4. Read the transaction details and Click "Deposit." If you have already deposited the funds, you can move to **Step 8**.
+5. To make sure your subscription is renewed every month, you need to deposit funds to the Spheron escrow wallet.
+6. Add the amount you want to approve and Click "Approve." This amount should be more than or equal to the plan's Token-Value (total amount).
+7. Add the amount you want to deposit and Click "Deposit." Make sure it is not more than the approved amount, or else the transaction will fail.
+8. Click "Upgrade Plan" to initiate the transaction. Upon successful transaction, the plan will be upgraded to Pro.
```
Comment (rekpero) [inline pages/billing/credits.mdx:38]:
```suggestion
8. Click "Upgrade Plan" to initiate the payment. Upon successful deduction from credit, the plan will be upgraded to Pro.
```

**Review by rekpero — CHANGES_REQUESTED** [review]
bro don't you read the content you are copying and also are you not trying it yourself, why is the whole context messed up? You should try the feature before you should write the steps around it, a lot of improvement has been done from last time the steps are now not the same, you should check properly what we should write instead of copying it blindly

File: pages/billing/credits.mdx (hunk 5 above)
Comment (aayushmahapatra) [reply pages/billing/credits.mdx:54]: We should remove the `Add Add-on Field` button also.

File: pages/billing/credits.mdx [hunk 12]
```diff
@@ -0,0 +1,55 @@
...
+  **NOTE:** <br /> Each credit has an expiry. Once your credit expires, you will
+  lose access to all the bonuses purchased with those credits.
+</Callout>
+
+## Upgrading Plans with Credits
+
+1. Navigate to the "My Plan" section under the **Billing** tab.
+2. Click "Change or upgrade plan" and Select your desired plan.
+3. Choose **Spheron Credit** in Select Payment Method.
+4. Read the transaction details and Click "Upgrade Plan" to initiate the payment. Upon successful deduction from credit, the plan will be upgraded to Pro.
```
Comment (rekpero) [inline pages/billing/credits.mdx:34]:
```suggestion
4. Read the payment details and Click "Upgrade Plan" to initiate the payment. Upon successful deduction from credit, the plan will be upgraded to Pro.
```

**Review by rekpero — CHANGES_REQUESTED** [review]

**Review by rekpero — APPROVED** [review]

**Pull request merged**